components/actions/blocks/action-dependencies/ActionDependenciesBlock.tsx

```diff
@@ -106,8 +141,15 @@ export function mapActionToDependencyGroups(
     []
   );
   const uniqueActions = uniqBy(flatDependencyList, 'id');
+  let expandedActions = uniqueActions;
+  if (getFullAction != null) {
+    expandedActions = uniqueActions.map((a) =>
+      Object.assign({}, getFullAction(a.id), a)
+    );
+  }
```

Comment:
Nitpick: preferring `const` over `let` is accepted as best practice in JS lore, instead we could do:

```suggestion
  const expandedActions = getFullAction
    ? uniqueActions.map((a) => Object.assign({}, getFullAction(a.id), a))
    : uniqueActions;
```